code/controllers/subsystems/zcopy.dm

```diff
@@ -135,14 +135,20 @@ SUBSYSTEM_DEF(zcopy)
 		// Figure out how many z-levels down we are.
 		var/depth = 0
 		var/turf/Td = T
+		var/overwrite_override = FALSE
 
 		while (Td.below)
 			Td = Td.below
+			if(!(Td.z_flags & ZM_MIMIC_OVERWRITE))
+				overwrite_override = TRUE
 
 		T.z_depth = depth = min(T.z - Td.z, OPENTURF_MAX_DEPTH)
 
 		var/t_target = OPENTURF_MAX_PLANE - depth	// this is where the openturf gets put
 
+		if(overwrite_override)
+			t_target = OPENTURF_MAX_PLANE - 1
```

Comment:
What is this for?

Reply:
I forgot I made this hack cause zcopy isn't doing the planes in the correct order or something

Reply:
Underlying zcopy issues are currently being addressed, holding off on this until that's done.